website/organizations/index.html

```diff
@@ -34,19 +34,19 @@ <h3 class="subheader-section">
         <h3 class="subheader-section">
           {{ _('Get Thunderbird for your organization') }}
         </h3>
-        <p class="font-lg tracking-wider leading-loose mb-10 mt-0 p-links-blue">
+        <p class="font-lg tracking-wider leading-loose mt-0 p-links-blue">
           {{ _('Thunderbird is available in a <a href="{syslang}">variety of languages</a> for all major operating systems (Windows / MacOS / Linux). For multi-lingual environments, you can download language pack add-ons and switch Thunderbird’s user interface language on the fly. We offer a number of resources specifically for organizations:')|f(syslang=url('thunderbird.latest.all')) }}
         </p>
 
-        <ul class="font-md tracking-wider leading-loose mb-6 p-links-blue">
+        <ul class="font-lg tracking-wider leading-loose mt-0 mb-6 p-links-blue">
           <li>
             {{ _('For Windows deployment, you can find the Thunderbird MSI Installer on the <a href="{syslang}">Systems &amp; Languages page</a>.')|f(syslang=url('thunderbird.latest.all')) }}
           </li>
           <li>
             {{ _('For more information about deploying and configuring Thunderbird in business and enterprise environments, please visit our enterprise deployment portal: <a href="{thunderbird_etn}">enterprise.thunderbird.net</a>.')|f(thunderbird_etn=url('thunderbird.enterprise.documentation')) }}
           </li>
           <li>
-            {{ _('You can also join the <a href="{tb_list_enterprise}">Thunderbird Enterprise mailing list on Topicbox</a> to connect with other administrators and share advice on Thunderbird deployment.')|f(tb_list_enterprise=url('thunderbird.lists.enterprise')) }}
+            {{ _('You can also join the <a href="{tb_list_enterprise}">Thunderbird Enterprise mailing list</a> on Topicbox to connect with other administrators and share advice on Thunderbird deployment.')|f(tb_list_enterprise=url('thunderbird.lists.enterprise')) }}
```

Comment:
Since we're using Topicbox we want to keep the name in the link as to avoid potential confusion if the user expects a link to a more "standard" mailing list.